packages/components-library/src/components/filters/ChangeFilters.vue

```diff
@@ -149,23 +141,27 @@ export default Vue.extend({
       this.$emit('input', this.selected)
     }
   }
-})
+}
 </script>
 
-<style scoped>
-.remove-button {
-  transition: color 0.2s;
-  height: inherit;
-  width: 1.5em;
-  text-align: center;
-  display: inline-block;
-  position: absolute;
-  right: 10px;
-  cursor: pointer;
-}
-.remove-button:hover {
-  color: var(--danger);
+<style lang="scss">
```

Comment:
why remove the scoped ? is this needed to use rollup and scss ?

Reply:
Reverting